doc/source/notebooks/models/load_model.ipynb

```diff
@@ -0,0 +1,356 @@
+       "          (v_proj): Linear(input_dim=4096, output_dim=1024, bias=False, init_fn=init_qkv_projection)\n",
+       "          (pos_encoder): RotaryEncoder(encoding_dim=128, max_seq_len=131072)\n",
+       "          (sdpa): TorchSDPA(attn_dropout_p=0.1)\n",
+       "          (output_proj): Linear(input_dim=4096, output_dim=4096, bias=False, init_fn=init_output_projection)\n",
+       "        )\n",
+       "        (self_attn_norm): None\n",
+       "        (self_attn_dropout): None\n",
+       "        (self_attn_residual): StandardResidualConnect()\n",
+       "        (encoder_decoder_attn): None\n",
+       "        (encoder_decoder_attn_dropout): None\n",
+       "        (encoder_decoder_attn_residual): None\n",
+       "        (encoder_decoder_attn_layer_norm): None\n",
+       "        (ffn_layer_norm): RMSNorm(normalized_shape=(4096,), eps=1E-05, elementwise_affine=True)\n",
+       "        (ffn): GLUFeedForwardNetwork(\n",
+       "          model_dim=4096, inner_dim_scale=0.666667, inner_dim_to_multiple=1024\n",
+       "          (gate_proj): Linear(input_dim=4096, output_dim=14336, bias=False)\n",
+       "          (gate_activation): SiLU()\n",
+       "          (inner_proj): Linear(input_dim=4096, output_dim=14336, bias=False)\n",
+       "          (inner_dropout): Dropout(p=0.1, inplace=False)\n",
+       "          (output_proj): Linear(input_dim=14336, output_dim=4096, bias=False)\n",
+       "        )\n",
+       "        (ffn_dropout): None\n",
+       "        (ffn_residual): StandardResidualConnect()\n",
+       "      )\n",
+       "    )\n",
+       "    (layer_norm): RMSNorm(normalized_shape=(4096,), eps=1E-05, elementwise_affine=True)\n",
+       "    (dropout): Dropout(p=0.1, inplace=False)\n",
+       "  )\n",
+       "  (final_proj): Linear(input_dim=4096, output_dim=128256, bias=False, init_fn=init_final_projection)\n",
+       ")"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from fairseq2.models.llama import get_llama_model_hub, create_llama_model\n",
+    "\n",
+    "model_hub = get_llama_model_hub()\n",
+    "model_config = model_hub.load_config(\"llama3_1_8b_instruct\")  # use llama3.1 8b preset as an example\n",
+    "\n",
+    "llama_model = create_llama_model(model_config)\n",
+    "llama_model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To check what are the registered models, we can leverage the `asset_store` in our runtime context."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from fairseq2.context import get_runtime_context\n",
+    "context = get_runtime_context()\n",
+    "asset_store = context.asset_store"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
```

Comment:
I think we DONT want to share our clusters info here ?it will not be relevant for general purpose usage